CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## edge-19.9.3
+
+* Helm
+  * Allow disabling namespace creation during install (thanks @KIVagant!)
+* Controller
+  * Fixed proxy injector timeout during a large number of concurrent injections
+  * Separated the destination controller into its own separate pod
+  * Updated Prometheus config to keep only needed cadvisor metrics
```

Comment:
this is okay, but maybe clearer wrt user impact as: "Updated Prometheus config to keep only needed cadvisor metrics, substantially reducing the number of time-series stored in most clusters"